app/models/integration_collection_mappings/base_collection_mapping.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module IntegrationCollectionMappings
+  class BaseCollectionMapping < ApplicationRecord
+    include PaperTrailTraceable
+    include SettingsStorable
+
```

Comment:
Can we add please:
`validates :mapping_type, uniqueness: { scope: :integration_id }`